physical_properties/pKa/analysis/analysis_of_typeIII_predictions/analysis_outputs_closest/StatisticsTables/statisticsLaTex/statistics.tex

```diff
@@ -17,40 +17,40 @@
+ np6b4 &    EC-RISM/B3LYP/6-311+G(d,p)-P2-phi-noThiols-2par &  1.938 [1.215, 2.777] &  1.435 [1.046, 1.967] &   -0.467 [-1.084, 0.298] &  0.709 [0.595, 0.866] &   1.083 [0.821, 1.482] \\
+ w4iyd &            Vertical scheme for type III submission &  1.939 [1.488, 2.336] &  1.578 [1.149, 2.015] &     1.211 [0.682, 1.797] &  0.849 [0.716, 0.927] &   1.256 [1.016, 1.454] \\
+ pwn3m &                                     Analog\_search &  1.970 [0.779, 2.881] &  1.115 [0.574, 1.832] &    0.285 [-0.382, 1.106] &  0.354 [0.013, 0.903] &   0.583 [0.073, 1.055] \\
+ f0gew &  EC-RISM/B3LYP/6-311+G(d,p)-P3NI-phi-noThiols-2par &  2.184 [1.351, 2.907] &  1.578 [1.095, 2.141] &   -0.733 [-1.511, 0.014] &  0.769 [0.661, 0.893] &   1.291 [1.026, 1.655] \\
+ xikp8 &  Direct scheme with single point correction for... &  2.340 [1.882, 2.732] &  2.026 [1.581, 2.468] &     0.933 [0.072, 1.671] &  0.867 [0.773, 0.932] &   1.524 [1.288, 1.795] \\
+ 5nm4j &        Substructure matches from experimental data &  2.450 [1.428, 3.336] &  1.583 [0.946, 2.336] &    0.046 [-0.788, 0.982] &  0.192 [0.001, 0.704] &  0.398 [-0.070, 0.821] \\
+ ad5pu &    EC-RISM/B3LYP/6-311+G(d,p)-P3NI-q-noThiols-2par &  2.508 [1.620, 3.341] &  1.744 [1.169, 2.431] &   -0.526 [-1.352, 0.383] &  0.726 [0.604, 0.852] &   1.373 [1.053, 1.774] \\
+ 0hxtm &                                 COSMOtherm\_FINE17 &  2.638 [0.965, 3.873] &  1.423 [0.714, 2.419] &    0.736 [-0.078, 1.884] &  0.127 [0.000, 0.799] &  0.406 [-0.185, 1.047] \\
+ ds62k &      EC-RISM/MP2/6-311+G(d,p)-P3NI-q-noThiols-2par &  2.987 [1.448, 4.642] &  1.883 [1.222, 2.918] &   -0.230 [-1.493, 0.751] &  0.540 [0.242, 0.880] &   1.171 [0.950, 1.400] \\
+ ttjd0 &           EC-RISM/MP2/cc-pVTZ-P2-phi-noThiols-2par &  2.989 [1.300, 4.674] &  1.695 [1.049, 2.661] &   -0.773 [-1.782, 0.108] &  0.516 [0.231, 0.884] &   1.147 [0.967, 1.347] \\
+ mpwiy &         EC-RISM/MP2/cc-pVTZ-P3NI-phi-noThiols-2par &  3.007 [1.479, 5.086] &  1.841 [1.189, 2.894] &   -0.491 [-1.660, 0.449] &  0.547 [0.241, 0.873] &   1.192 [0.952, 1.415] \\
+ nb004 &    EC-RISM/MP2/6-311+G(d,p)-P3NI-phi-noThiols-2par &  3.277 [1.470, 4.981] &  1.985 [1.231, 2.979] &   -0.102 [-1.344, 0.874] &  0.523 [0.254, 0.879] &   1.296 [1.042, 1.576] \\
+ nb003 &         EC-RISM/MP2/6-311+G(d,p)-P3NI-phi-all-2par &  3.288 [1.474, 4.993] &  1.997 [1.245, 2.966] &   -0.138 [-1.322, 0.974] &  0.526 [0.241, 0.875] &   1.305 [1.044, 1.597] \\
+ nb005 &           EC-RISM/MP2/6-311+G(d,p)-P2-phi-all-1par &  4.111 [1.864, 6.547] &  2.443 [1.512, 3.884] &   -0.470 [-2.121, 0.901] &  0.486 [0.176, 0.881] &   1.435 [1.116, 1.757] \\
+ nb006 &         EC-RISM/MP2/6-311+G(d,p)-P3NI-phi-all-1par &  4.558 [2.431, 7.052] &  2.999 [1.985, 4.609] &   -0.370 [-2.117, 1.164] &  0.517 [0.219, 0.879] &   1.642 [1.316, 2.021] \\
```

Comment:
I'm confused about why the statistics are changing, could you explain what changed in the analysis?

Reply:
@MehtapIsik

Reply:
I think this difference in confidence intervals is because of a change in the number of bootstrap samples from 10000 to 1000. I plan to correct it back to using 10000 samples, but I can't do it at the moment. Something else is broken in my analysis script.